backend/src/main/java/com/github/hirnstromwelle/spritebranch/controllers/ItemController.java

```diff
@@ -30,4 +31,13 @@ public ItemDto createItem(@Valid @RequestBody ItemDto itemDTO) {
         Item createdItem = itemService.createItem(item);
         return modelMapper.map(createdItem, ItemDto.class);
     }
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> deleteItem(@PathVariable String id) {
+        try {
+            itemService.deleteItem(id);
+            return ResponseEntity.noContent().build();
+        } catch (Exception e) {
+            return ResponseEntity.notFound().build();
+        }
```

Comment:
Unnötig. Du kannstz einfach die delete methode aus dem Service aufrufen ohne try catch. Auch brauchst du kein ResponseEntity. Du kannst den Endpunkt mit @ResponseStatus annotieren. Sollte in Fehler auftreten, sollte er im service geschmissen werden und dann über einen ExceptionHandler an den User rausgegeben werden